Fixes "Adds digitigrade jumpsuit alts using ThatGuyPhil's icons #11744"

nmajask opened the pull request:
# Github documenting your Pull Request

So there were a few is issues with PR #11744. For example, the digitigrade alts only worked on a few jumpsuits, footwraps wouldnt look good with them, and it was coded very poorly, this PR attempts to fix many of the issues as well as some minor stuff in my other PRs.

I removed the digiversion and digiadjusted vars from any file they appear in, as they only serve to do the same thing as mutantrace_variation. The main reason I had these was so you could have a digialt for a jumpsuit but not have it break if you dont have one for the adjusted version, but this would result in you having to define 3 vars to adding alts to jumpsuits.

I added alts for a ton of other jumpsuits, almost all of which were functional in my test server when I created the PR but didnt make it in because I forgot to upload the files. Now all job uniforms should have digialt. Also added digialts for the explorer suit and wardens jacket and changed the icons for footwraps so that they actually cover digitigrade feet.

Finally, I cleaned ups some of the stuff from #11528 and #11542. They are just unused icons and commented out things.

# Wiki Documentation

I dont think anything on the wiki will be affected by this as its mostly code stuff, but I could be wrong.

# Changelog
:cl:  
rscadd: Added many more digitigrade jumpsuit alts and two suit ones
tweak: tweaked the footwrap icons so they cover digitigrade feet
tweak: cleaned some code
/:cl:

nmajask committed:
Fixes some stuff

- Removed digiversion and digiadjusted vars from _under.dm and any file they appear in as they are redundant
- Added support to medsci, engi, security, and artist jumpsuits for digialts which I had finished but forgot to put into the PR... oops
- Added digialts for wardens jacket and explorer's suit
- Changed the footwrap icon so it fits digi feet. Only temperary until someone makes a resprite
- Cleaned up some of my previous PRs

nesisknesis reviewed (approved):
can't go a week without jamie merging unverified art prs

nmajask committed:
Removes the footwrap changes

- Removes the footwrap as Footwrap re-sprite #11885 by
cuackles has way better sprites than mine and to prevent a merge conflict with it. If that PR doesn't get merged, ill re-add the changes.

nmajask committed:
Fixes chaplains jumpsuit

- Fixes chaplain's jumpsuit so it doesn't use the placeholder
